Source/TheLudoGameEngine/Classes/Engine.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace TheLudoGameEngine
+{
+    public class Engine : Game
+    {
+        MyContext myContext = new MyContext();
```

Comment:
This gives a very high coupling to MyContext.
Maybe you should also find a more meaningful name like LudoContext